staff/staff_views_helper.py

```diff
@@ -79,18 +80,46 @@ def add_volunteers_and_students_to_classroom(request, form, classroom):
         django_user = UserProfile.objects.get(
             salesforce_id=volunteer.client_id.lower()
         ).user_id
+        mark_user_in_classroom(django_user)
         classroom.volunteers.add(django_user)
         email_list.append(DjangoUser.objects.get(id=django_user).email)
     for student in form.cleaned_data.get("students"):
         enroll_in_class(form, student)
         django_user = UserProfile.objects.get(
             salesforce_id=student.client_id.lower()
         ).user_id
+        mark_user_in_classroom(django_user)
         classroom.students.add(django_user)
         email_list.append(DjangoUser.objects.get(id=django_user).email)
     email_classroom(request, email_list, classroom.course)
 
 
+def mark_user_in_classroom(user):
```

Comment:
This seems a bit strange to me, what is the purpose of this flag on the UserProfile? This can all be computed by querying the Classroom and related tables and can't it?

Reply:
Yes, along with using the correct reference dates.  I didn't even follow-through on intended usage because I realized we didn't need it, and then forgot to delete it.  Taken care of in 35858ae5613e038813d6ac6c2383bc679359b3dd